From b60357b664b11cf25aee797cc41add641e1ff15e Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Fri, 30 Aug 2019 16:54:15 +0800
Subject: [PATCH 067/245] Revert "ptp: move DPAA2 PTP config to
 fsl-dpaa2/Kconfig"

commit 8aca645a19ac1d0abb2f27305844f6fb55115509 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

This reverts commit 68177e7da0ed7c0ff5ded0139baee57d3f421939.

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/ptp/Kconfig                | 14 ++++++++++++++
 drivers/staging/fsl-dpaa2/Kconfig  | 13 -------------
 drivers/staging/fsl-dpaa2/Makefile |  2 +-
 3 files changed, 15 insertions(+), 14 deletions(-)

diff --git a/drivers/ptp/Kconfig b/drivers/ptp/Kconfig
index 53e31d74502b..0faa6d8c468b 100644
--- a/drivers/ptp/Kconfig
+++ b/drivers/ptp/Kconfig
@@ -55,6 +55,20 @@ config PTP_1588_CLOCK_QORIQ
 	  To compile this driver as a module, choose M here: the module
 	  will be called ptp_qoriq.
 
+config PTP_1588_CLOCK_DPAA2
+	tristate "Freescale DPAA2 as PTP clock"
+	depends on FSL_DPAA2_ETH
+	select PTP_1588_CLOCK
+	default y
+	help
+	  This driver adds support for using the DPAA2 1588 timer module
+	  as a PTP clock. This clock is only useful if your PTP programs are
+	  getting hardware time stamps on the PTP Ethernet packets
+	  using the SO_TIMESTAMPING API.
+
+	  To compile this driver as a module, choose M here: the module
+	  will be called dpaa2-rtc.
+
 config PTP_1588_CLOCK_IXP46X
 	tristate "Intel IXP46x as PTP clock"
 	depends on IXP4XX_ETH
diff --git a/drivers/staging/fsl-dpaa2/Kconfig b/drivers/staging/fsl-dpaa2/Kconfig
index f65d348b2c59..e756916c2772 100644
--- a/drivers/staging/fsl-dpaa2/Kconfig
+++ b/drivers/staging/fsl-dpaa2/Kconfig
@@ -52,19 +52,6 @@ config FSL_DPAA2_ETH_DCB
           (PFC) in the driver.
 
           If unsure, say N.
-
-config FSL_DPAA2_PTP_CLOCK
-	tristate "Freescale DPAA2 as PTP clock"
-	select PTP_1588_CLOCK
-	default y
-	help
-	  This driver adds support for using the DPAA2 1588 timer module
-	  as a PTP clock. This clock is only useful if your PTP programs are
-	  getting hardware time stamps on the PTP Ethernet packets
-	  using the SO_TIMESTAMPING API.
-
-	  To compile this driver as a module, choose M here: the module
-	  will be called dpaa2-rtc.
 endif
 
 source "drivers/staging/fsl-dpaa2/mac/Kconfig"
diff --git a/drivers/staging/fsl-dpaa2/Makefile b/drivers/staging/fsl-dpaa2/Makefile
index 8c3549d72876..a921051f0a79 100644
--- a/drivers/staging/fsl-dpaa2/Makefile
+++ b/drivers/staging/fsl-dpaa2/Makefile
@@ -5,5 +5,5 @@
 obj-$(CONFIG_FSL_DPAA2_ETH)		+= ethernet/
 obj-$(CONFIG_FSL_DPAA2_MAC)	+= mac/
 obj-$(CONFIG_FSL_DPAA2_EVB)	+= evb/
-obj-$(CONFIG_FSL_DPAA2_PTP_CLOCK) += rtc/
+obj-$(CONFIG_PTP_1588_CLOCK_DPAA2) += rtc/
 obj-$(CONFIG_FSL_DPAA2_ETHSW)	+= ethsw/
-- 
2.17.1

